src/App/App.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './App.css';
+import Header from './../Components/Header/header'
```

Comment:
Consider removing "./" part from the import string - it is redundant.

Reply:
i've removed the excess "./" part
![image](https://user-images.githubusercontent.com/46630057/74604781-3756a180-50d2-11ea-9f20-c070a6384f69.png)